src/vim9expr.c

```diff
@@ -811,7 +873,34 @@ compile_load(
 	    }
 	}
 	if (gen_load)
-	    res = generate_LOAD(cctx, ISN_LOAD, idx, NULL, type);
+	{
+	    // Optimization: check if can generate LOAD CLASSMEMBER.
+	    int did_classmember = FALSE;
+	    if (type->tt_type == VAR_OBJECT
+		&& (type->tt_class->class_flags & CLASS_INTERFACE) == 0
+		&& !gen_load_outer)
+	    {
+		// Got an object that is not an interface.
+		int is_class;
+		char_u *name_end;
+		ocmember_T *m;
+		int idx01;
```

Comment:
Usually numbers are not used in variable names in the Vim source code.

Reply:
If there's an `index`, and you want a temporary index, how is that typically named? `tindex`?

Reply:
In this case, can you use 'member_index' or 'cm_idx'?

Reply:
> In this case, can you use 'member_index' or 'cm_idx'?

Nice thing about idx01, is that it's obviously short term local var. In a large function, and vim has some monsters, that can be useful. Anyway, I got it.

Reply:
Vim source does have quite a few large functions.  Over a period of several years, I have refactored many large functions and files.  But still we have a long way to go in terms of improving the code.